sdk/src/main/java/io/dapr/PostMapping.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Documented
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface PostMapping {
```

Comment:
PostMapping belongs to Spring, not Dapr.

Reply:
Hmmm, why did I think I needed this? Guess I was missing an include?